Geo3x3.php

```diff
@@ -60,4 +67,3 @@ public static function decode(string $code): array {
     return [$lat, $lng, $level, $unit];
   }
 }
-?>
```

Comment:
PHP scripts that do not output HTML should not have `?>` closing tags to prevent extra output.